backend/tilavarauspalvelu/models/recurring_reservation/queryset.py

```diff
@@ -44,6 +44,10 @@ def requiring_access_code(self) -> Self:
             )
         ).filter(has_missing_access_codes=True)
 
+    def has_active_access_code(self) -> Self:
+        """Return all recurring reservations should have an active access code."""
+        return self.filter(L(should_have_active_access_code=True))
```

Comment:
Method name is not accurate to its function 🤔 
From the name I would assume that it filters series that HAVE an active access code, but really it filters series that SHOULD HAVE an active access code, but might not necessarily be active

Reply:
Changed name.